### ISSUE TITLE
Print training summary metrics after training

### ISSUE THREAD

**Issue by gregorsternat** [Epitech-2nd-Year-Projects/my_torch#32]
At the end of `--train`, print a concise summary of training:

* Final training and validation loss.
* Final training and validation accuracy.
* Number of epochs performed (taking early stopping into account).

This will be reused later for benchmarks and documentation.
